ArduCopter/autoyaw.cpp

```diff
@@ -154,9 +154,7 @@ void Mode::AutoYaw::set_roi(const Location &roi_location)
         auto_yaw.set_mode_to_default(false);
 #if HAL_MOUNT_ENABLED
         // switch off the camera tracking if enabled
-        if (copter.camera_mount.get_mode() == MAV_MOUNT_MODE_GPS_POINT) {
```

Comment:
Why remove this qualifier?

Reply:
I think it is OK to remove this qualifier because it is redundant.  The new "clear_roi_target()" has almost exactly the same code. https://github.com/ArduPilot/ardupilot/blob/master/libraries/AP_Mount/AP_Mount_Backend.cpp#L96

Reply:
As @rmackay9 said, `mount..clear_roi_target()` does the same thing, _and_ sets some internal flags within `AP_Mount`.